plugins/analytics-provider-segment/src/plugin.ts

```diff
@@ -16,10 +16,9 @@
 import { createPlugin } from '@backstage/core-plugin-api';
 
 /**
- * @deprecated Importing and including this plugin in an app has no effect.
- * This will be removed in a future release.
  *
- * @public
+ * @public Importing and including this plugin in an app has no effect.
+ * This will be removed in a future release.
```

Comment:
This won't be removed in future release 🙂

Reply:
my bad :sweat_smile: